tutorials/data-lake-file-containers-hdlfscli/data-lake-file-containers-hdlfscli.md

```diff
@@ -128,14 +132,14 @@ Verify the certificate was signed by the CA.
 
 To obtain the subject string of a certificate in the RFC2253 format used in HDL Files authorizations (omit the "subject="prefix).
 
-**Note:** you will need this later when you configure authentication for HDL Files.
+**Note:** You will need this later when you configure authentication for HDL Files.
```

Comment:
Please don't start a note with the label "Note". The formatting already makes it a note. 
Please check the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9#mcetoc_1dtqfqchq21) for details.